docs/specification/unique-identifiers.md

```diff
@@ -48,11 +48,13 @@ Field name | Value type | Description
 
 ##### Constraints
 
-Field name  | Relationship | Field Name
+Field name  | Relationship | Description
 ------------|:------------:|------------
-len(`uids`) | <=           | 8
-len(`scope`)| <=           | 128, UTF-8 encoded
-len(`id`)   | <=           | 512, UTF-8 encoded
+len(`uids`) |      <=      | 8
+len(`scope`)|      <=      | 128, UTF-8 encoded
+len(`id`)   |      <=      | 512, UTF-8 encoded
+`scope`     |              | cannot include "::"
```

Comment:
Not certain that what's written there is a `Description`, though it's definitely not a `Field Name`.  
```suggestion
`scope`     |              | cannot include `::`
```